CONTRIBUTING.rst

```diff
@@ -186,15 +186,48 @@ WMCore defines its own pylintrc `here <https://github.com/dmwm/WMCore/blob/maste
 Thus, when evaluating your changes, please run pylint by passing this pylintrc file in the command line, your code should get scored 8 or above.
 Unless there is a very good reason, we discourage the use of pylint disable statements.
 
+Project Docstrings Best Practices
+---------------------------------
+
+With the goal of uniformizing and making the project more readable, we are adopting the `Sphinx` docstring style, which uses
```

Comment:
In addition to that I suggest to provide script(s) to generate `Sphinx` doc locally, this may be helpful to test docstring before commits. Ideally, it should be part of CI/CD, sort of similar to `pylint` which developers can run locally w/o waiting jenkins (and this case `Sphinx`) to generate docs. This may help to spot and debug issues earlier.

Reply:
My initial goal with this brief documentation is not to run an auto documentation setup, but to actually avoid free will and different styles when writing docstrings out. It's annoying to look at a module and see 2, 3 or even more styles of docstrings.

This suggestion to add it to our CI and also have the necessary tools in our wmcore docker image is very good though. However, I think it deserves its own ticket. Honestly, I don't even know if we would be able to run an autodoc setup while we have all sorts of standards and styles in the docstrings...